lxd/db/storage_pools.go

```diff
@@ -109,22 +105,22 @@ SELECT storage_volumes.name, storage_volumes.type, projects.name, storage_volume
 		return nil, err
 	}
 
+	appendUrl := func(r vol, u *api.URL, project string) {
```

Comment:
As the project logic isn't needed, you probably dont need the project arg here and can just add it outside.